### DISCUSSION TITLE
So, ya I lost the jks/key-signing again.

### DISCUSSION THREAD

**Discussion by Iamlooker** [Droid-ify/client#153] [Announcements]
My laptop got atolen a while back and I got it back pretty soon, but the data was wiped and as a stupid head, which I am, I hadn't backed-up my jks key anywhere. I only noticed this while I was creating a repease build yesterday in the night.

So this means the next update will need to be reinstalled, I am sorry for the inconvenience.

**Comment by kevin0t** [comment]
we'll need to download the apk from github and install it after uninstalling previous  build right ?

will there be update notification in droid-ify itself ? i guess updating won't work , but still would notify us that an update is available.

**Reply by Iamlooker** [reply]
Yes if you are using GitHub Release or Izzy4Droid Release. FDroid release will be untouched
